project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -89,6 +97,7 @@
             </xpath>
             <field name="partner_ids" position="after">
                 <field name="resource_type" attrs="{'readonly': [('is_task_event','=',True)]}"/>
+                <field name="sector_id" attrs="{'readonly': [('is_task_event','=',True)]}"/>
```

Comment:
@jananjoy same comment (the sector field must be added in the project_event/views/calendar_event_category_view.xml)

Reply:
@rimbendhaou Done